wurst/systems/units/hostiles/trolls/Observer.wurst

```diff
@@ -20,7 +20,13 @@ public class Observer extends Scout
 
     override function getTypeId() returns int
         return unitTypeId
+        
+    override function postCreate()
+        super.postCreate()
 
+        getUnit().removeAbility(SPELL_PING_ENEMY)
+        getUnit().addAbility('XSGR')
+        //getUnit().addAbility(SPELL_OBS_WARD_AREA_ID)
```

Comment:
Why is this commented?

Reply:
Ward area is a hero ability, I believe adding hero ability doesn't work this way, so I put it on GUI, didn't know how to do it otherwise and forgot to delete this.